packages/common-all/src/drivers/string2Note.ts

```diff
@@ -0,0 +1,50 @@
+import matter from "gray-matter";
+import YAML from "js-yaml";
+import _ from "lodash";
+import { DNodeUtils } from "../dnode";
+import { DVault } from "../types";
+import { genHash } from "../utils";
+
+/**
+ * NOTE: Temporarily duplicated from common-server/filesv2.ts to get Dendron Web
+ * Extension working
+ * @param calculateHash - when set, add `contentHash` property to the note
+ *  Default: false
+ * @returns
+ */
+export function string2Note({
```

Comment:
can we add a comment in this function in `common-server` to port over any changes we make to that function?

Reply:
I just deleted the common-server one and updated imports.  Code between this and common-server one were identical.